test/compression.js

```diff
@@ -676,6 +676,35 @@ describe('compression()', function () {
       .end()
     })
   })
+
+  describe('when callbacks are used', function () {
+    it('should call the passed callbacks', function(done){
+      var callbacks = 0;
+      var server = createServer({ threshold: '1kb' }, function (req, res) {
+        res.setHeader('Content-Type', 'text/plain')
+        res.write('Hello', null, function(){
+          callbacks++
+          res.flush()
```

Comment:
Since the test is labeled "when callbacks are used should call the passed callbacks", it doesn't sound like the test should be testing `.flush()` functionality. Is there a reason to call `.flush()` in theses callbacks? If it is testing something, should that be a different test, or can it be reflected in the name?